src/platform/posix.c

```diff
@@ -72,8 +72,13 @@ void platform_get_date_utc(rct2_date *out_date)
 	time(&rawtime);
 	timeinfo = gmtime(&rawtime);
 	out_date->day = timeinfo->tm_mday;
-	out_date->month = timeinfo->tm_mon;
-	out_date->year = timeinfo->tm_year;
+	if (OPENRCT2_PLATFORM == "Windows") {
```

Comment:
Well apparently `posix.c` is also used by Windows. Otherwise I don't know how my fix solved @mrtnptrs problem on his Windows machine.

Perhaps we can move this code to `shared.c` if you want to be strict about it.